test/Python/src/TuplePrint.lf

```diff
@@ -1,5 +1,5 @@
-// Source produces a statically allocated array, which it passes
-// to Print. The destination references the array directly.
+// Source produces a tuple, which it passes to Print.
```

Comment:
`ArrayAsType` is a common test. Renaming it to `TuplePrint` causes the test framework to complain about the absence of this test. If you decide to keep the name, I think it should be moved to `test/Python/src/target`.

Reply:
I think I'll just rename it back and keep a tab on it